scripts/run_backend_tests.py

```diff
@@ -383,18 +384,22 @@ def main(args=None):
             include_load_tests=include_load_tests)
 
     # Prepare tasks.
+    concurrent_count = min(multiprocessing.cpu_count(), 24)
```

Comment:
Worth extracting 24 to a named constant?

Reply:
I agree, did that!